### ISSUE TITLE
Login failed with old mnemonic

### ISSUE THREAD

**Issue by bilthon** [Blockstream/green_android#102]
### Description

I'm just trying to recover an old wallet I created in another device re-entering the mnemonic. The wallet was empty, but I still should be able to recover it. Instead all I can see is the `id_login_failed` toast.

### Version

3.4.2

### Steps to reproduce

I just enter the mnemonic and get that error.

### Expected behaviour

I should be able to recover my wallet.

### Actual behaviour

I just get a toast saying `Login Failed`

### Screenshots

I can't take a screenshot of this activity, since it is protected.

### Device or machine

Nexus 5, Android 6.0.1

**Comment by jgriffiths** [comment]
Are you able to log in on any other instance of the wallet (green_qt for example?)

**Comment by bilthon** [comment]
No, I actually downloaded green_qt to try that out, but backed off because of this permission request.

![blockstream_qt](https://user-images.githubusercontent.com/1840721/98626295-67d51000-22df-11eb-8427-ea3720f77337.png)

I know this is not an issue from this product, but I just don't understand why that app would need that. So that prevented me from going further with this test. I might give it a try on another machine.

**Comment by jgriffiths** [comment]
@bilthon Please raise a bug at https://github.com/Blockstream/green_qt/issues about the permission issue.

Did you have a watch only username or 2fa set up on this wallet? If so please email a link to this issue to info@greenaddress.it and the details (not any passwords!) so we can help further. In the absence of any further information I have to assume that the wallet was either deleted by you or the mnemonic is wrong.

If you can try on another machine please update here with the results, thanks.

**Comment by jgriffiths** [comment]
Also, do you know how old the wallet was?

**Comment by domegabri** [comment]
> No, I actually downloaded green_qt to try that out, but backed off because of this permission request.
> 
> ![blockstream_qt](https://user-images.githubusercontent.com/1840721/98626295-67d51000-22df-11eb-8427-ea3720f77337.png)
> 
> I know this is not an issue from this product, but I just don't understand why that app would need that. So that prevented me from going further with this test. I might give it a try on another machine.

We have introduced a bug when we added support for Ledger in the desktop app. This will be fixed in the next release (0.0.36) that is coming in a few days

**Comment by angelix** [comment]
@bilthon please reopen if needed.
